### ISSUE TITLE
Accessibility: Screen Readers (JAWS / NVDA)

### ISSUE THREAD

**Issue by victorsimonlee** [OfficeDev/microsoft-teams-apps-company-communicator#901]
Hello,

Accessibility tools (screen readers like **JAWS** and **NVDA**) are unable to read the cards sent by the Company Communicator. Both tools can identify that a "card message" was received but are unable to read the contents of the message itself.

_At the moment, it's not entirely clear to us if this is an issue with Company Communicator or MS Teams Adaptive Cards as a whole._

Also, by comparing the JSON generated by the Company Communicator vs what is available in the Adaptive cards specification [(adaptivecards.io)](https://adaptivecards.io/explorer/) or by using Teams Developer Portal (card design tool), I can see many properties were left out. 

We, in particular, would be interested in properties like:

- Image's "Alternate Text"
- "Speak"

Therefore, we would like to request the addition of the above noted properties to the Company Communicator card messages.

Thank you.

**Comment by v-royavinash** [comment]
@victorsimonlee , Thanks for highlighting the request. We will review it internally and get back to you with details. If screen reader support is not enabled we will add it to backlog for future releases.

**Comment by victorsimonlee** [comment]
Contacted Microsoft Enterprise Disability Answer Desk regarding MS Teams and Accessibility issues with Adaptive Cards. Their response:

_"Thank you for contacting the Microsoft enterprise Disability Answer Desk. 

Yes, we are aware, and the issue is on the Engineering team’s radar. Currently, there is no other way to access the content present inside the card. We will update you once we hear back an update from the engineering team."_

**Comment by gsv022** [comment]
@victorsimonlee  , This has been added to our backlog list and will be planning to implement it in future releases post the discussion with our internal team. For now, we are closing this request as we have provided the inputs for the query raised. If you face any issue with company communicator please raise a new issue for further assistance.

**Comment by glblinck** [comment]
I know this was closed as "completed", as this was added to the engineering backlog, but do we have any timeframe on when this might be worked on? There are some very large systems that have put using this solution on hold due to the lack of the cards working with screen readers such as JAWS.

**Comment by gsv022** [comment]
@victorsimonlee , Company Communicator has been made accessibility compliant starting from version 5.4.1 onwards. Please update to the latest version since the release has been done already and kindly provide us with your feedback.

**Comment by MNL79** [comment]
@glblinck Did upgrading to v5.4.1 enable the app to work with JAWS? Or is it still just seeing it as a card and not reading the message??
